main.cpp

```diff
@@ -105,6 +106,21 @@ struct FloatType
 
     operator float() const { return *a; }
 
+    FloatType& apply( std::function< FloatType&( float& )> func )
+    {
+        if (func != nullptr )
+            func( *a );
+        return *this;
+    }
+
+    using funcPtr = void(*)( float& );
```

Comment:
`using` directives follow the same rules as Type naming.
- [ ] Capitalize your alias.